### ISSUE TITLE
Unreal Championship: Intro then crash

### ISSUE THREAD

**Issue by PatrickvL** [Cxbx-Reloaded/game-compatibility#102]
_From @Nrw1337 on May 7, 2017 19:10_

Displays the intro logos correctly if vsync is forced otherwise the screen will flicker, then plays the wmv logo files in a small panel upper left on the screen. Finally, it reaches the main menu load splash screen and crashes.

Intro logo:

![screenshot_112](https://cloud.githubusercontent.com/assets/25694960/25784171/c134a770-3371-11e7-825f-31fce7367224.png)

Crash point:
![image](https://cloud.githubusercontent.com/assets/25694960/25784285/b0766788-3372-11e7-9957-bb79776c6001.png)

Crash:
![screenshot_113](https://cloud.githubusercontent.com/assets/25694960/25784172/c2cc2d10-3371-11e7-8e1d-d56829f99101.png)

Build: cd50eeba (May 7 2017)
Log: 
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/files/981868/KrnlDebug.txt)
[CxbxDebug.txt](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/files/981870/CxbxDebug.txt)

_Copied from original issue: Cxbx-Reloaded/Cxbx-Reloaded#451_

**Comment by PatrickvL** [comment]
_From @blueshogun96 on May 7, 2017 19:23_

It doesn't seem to like the way it handles mip map generation.  I'm remains a mystery to me...

Shogun

**Comment by hotzenplotz3000** [comment]
Tested PAL-Version with build 75b82422
still the same behavior, but bit different error:
![capture20170912185107686](https://user-images.githubusercontent.com/30478365/30340377-9c005cca-97f2-11e7-81f8-76de33462cfa.png)

[Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1296871/Xbe.txt)
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1296872/KrnlDebug.txt)
[hlecache.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1296873/hlecache.txt)

**Comment by PatrickvL** [comment]
@jarupxx @Radwolfie Please notice the reported crash addresses: XActEngCreate. Would this be resolved once your XACT OOVPA database work is merged into master?

**Comment by jarupxx** [comment]
Tested NTSC-J version, this issue will be resolved by our XACT OOVPA database work.
But still not work as infinite loop from Crash point.
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1298833/KrnlDebug.txt)

**Comment by LukeUsher** [comment]
There are some regressions but in general by branch is showing huge progress when compared to master, don't worry, work is being done to get it into a mergeable state

**Comment by T02my** [comment]
state-partially-working. It crashes only at CTF mode in single player

https://www.youtube.com/watch?v=oTLqFZV9vRE&t=68s

**Comment by blueshogun96** [comment]
I'll try that CTF map and see if it does the same thing for me.  I've been able to play and complete death matches without issue.  Even the DLC works for me, so far.

Shogun

**Comment by dsk80004** [comment]
Freezes at the same place it use to again. Turning of patches doesn't help.

**Comment by esnesi** [comment]
I see videos that people can actually reach the menu, and play the game on youtube on build v63bc65ec
[Video here](https://www.youtube.com/watch?v=nD5suLIjN48)
I am not able to replicate this due to:
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/3929403/KrnlDebug.txt)

**Comment by bryce13950** [comment]
In the most recent build, the opening fmv plays, but it hangs after that.

[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4672458/KrnlDebug.txt)

**Comment by bryce13950** [comment]
The title id is IG-036 v1.00

**Comment by ratkosrb** [comment]
It throws this error while the intro video plays, but if you don't press anything the video will continue to play until it's over.
![Untitled](https://user-images.githubusercontent.com/35845488/99719436-f3eff000-2ab4-11eb-9add-0fd94141a34e.jpg)

**Comment by mainzerbe** [comment]
Goes ingame now, playable. However, it sometimes hungs up if you leave it for longer than 10 seconds within a menu.

![uc](https://user-images.githubusercontent.com/66551412/100527451-e7654900-31d2-11eb-8686-19db216f0e0f.PNG)
![uc1](https://user-images.githubusercontent.com/66551412/100527454-ea603980-31d2-11eb-8edf-4d1260f176a2.PNG)
![UC2](https://user-images.githubusercontent.com/66551412/100527456-ec29fd00-31d2-11eb-9076-a972a9f2c5d1.PNG)
![uc3](https://user-images.githubusercontent.com/66551412/100527457-ecc29380-31d2-11eb-99d6-bb28ebfa10c4.PNG)
![uc4](https://user-images.githubusercontent.com/66551412/100527458-ed5b2a00-31d2-11eb-8e7a-2f2689f5a554.PNG)
![uc5](https://user-images.githubusercontent.com/66551412/100527459-edf3c080-31d2-11eb-8034-dedf1f5e991e.PNG)
